src/Http/CurlDispatcher.php

```diff
@@ -116,7 +116,16 @@ class_exists('Composer\\CaBundle\\CaBundle')
      */
     protected function exec(Url $url, array $options)
     {
-        $connection = curl_init((string) $url);
+        /*
+         * Not sure about this fix!
+         * The problem is that url to string produces encoded url params like this:
+         *  https://www.google.com/maps/%4070.8853928%2C-40.8414069%2C4z
+         * This url would fail.
+         * Instead this should be called:
+         * https://www.google.com/maps/@70.8853928,-40.8414069,4z
+         * */
+        // url could be encoded at this point, so decode it
+        $connection = curl_init(urldecode((string) $url));
```

Comment:
This change cause other errors, so it's better to fix this issue inside the Url class.
The path is encoded here: https://github.com/oscarotero/Embed/blob/master/src/Http/Url.php#L314
And for a similar issue, I've created this function: https://github.com/oscarotero/Embed/blob/master/src/Http/Url.php#L664 that restore the escaped double colon. So we can consider to unescape also the `@` and `,` characters. This function is applied only to the filename path, but I think it can be applied to the full path.